models/federation.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+// Following is a single following record.
+// Technically it's the same as Followers but we keep it separate
```

Comment:
I think we can come up with a name that can be used in both cases, the same reason as i suggest to rename the file.

Reply:
so, like I had it to begin with. :)